x-pack/plugins/security_solution/public/detections/components/alerts_table/grouping_settings/group_stats.tsx

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiIcon } from '@elastic/eui';
+import React from 'react';
+import type {
+  BadgeMetric,
+  CustomMetric,
+} from '../../../../common/components/grouping/accordion_panel';
+import type { RawBucket } from '../../../../common/components/grouping';
+import * as i18n from '../translations';
+
+const getSingleGroupSeverity = (severity?: string) => {
+  switch (severity) {
+    case 'low':
+      return (
+        <>
+          <EuiIcon type="dot" color="#54b399" />
+          {i18n.STATS_GROUP_SEVERITY_LOW}
+        </>
+      );
+    case 'medium':
+      return (
+        <>
+          <EuiIcon type="dot" color="#d6bf57" />
+          {i18n.STATS_GROUP_SEVERITY_MEDIUM}
+        </>
+      );
+    case 'high':
+      return (
+        <>
+          <EuiIcon type="dot" color="#da8b45" />
+          {i18n.STATS_GROUP_SEVERITY_HIGH}
+        </>
+      );
+    case 'critical':
+      return (
+        <>
+          <EuiIcon type="dot" color="#e7664c" />
+          {i18n.STATS_GROUP_SEVERITY_CRITICAL}
+        </>
+      );
+  }
+  return null;
+};
+
+const multiSeverity = (
+  <>
+    <span className="smallDot">
+      <EuiIcon type="dot" color="#54b399" />
+    </span>
+    <span className="smallDot">
+      <EuiIcon type="dot" color="#d6bf57" />
+    </span>
+    <span className="smallDot">
+      <EuiIcon type="dot" color="#da8b45" />
+    </span>
+
+    <span>
+      <EuiIcon type="dot" color="#e7664c" />
+    </span>
+    {i18n.STATS_GROUP_SEVERITY_MULTI}
+  </>
+);
+
+export const getSelectedGroupBadgeMetrics = (
+  selectedGroup: string,
+  bucket: RawBucket
+): BadgeMetric[] => {
+  const defaultBadges = [
+    {
+      title: i18n.STATS_GROUP_ALERTS,
+      value: bucket.doc_count,
+      width: 50,
+      color: '#a83632',
+    },
+  ];
+  switch (selectedGroup) {
+    case 'kibana.alert.rule.name':
+      return [
+        {
+          title: i18n.STATS_GROUP_USERS,
+          value: bucket.usersCountAggregation?.value ?? 0,
+        },
```

Comment:
Super nit: looks like the badge value isn't centered when defaulting to `0`.

<img width="97" alt="image" src="https://user-images.githubusercontent.com/2946766/216445676-2fe8368c-18a2-49e5-a2eb-33566a35a97a.png">

Looks like there's an extra span in there keeping things centered when a value is present

<img width="772" alt="DevTools_-_localhost_5601_kbn_app_security_rules_id_a7bd4040-a337-11ed-947e-e5d337f16d85_alerts" src="https://user-images.githubusercontent.com/2946766/216444645-69341288-fdc9-4c18-ab63-d0d2860adf4c.png">